services/frontend/Makefile

```diff
@@ -0,0 +1,5 @@
+frontend: $(GOVUK_ROOT_DIR)/$@ content-store router static govuk-content-schemas publishing-api
```

Comment:
The first dependency should be `$(GOVUK_ROOT_DIR)/frontend`, the `$@` form doesn't work and got reverted.

Reply:
The makefile has changed again 🙀